### PR TITLE
Add Shelly RGBW2 pinout to sonoff.rst

### PR DESCRIPTION
**Pull request by erdii**
## Description:

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#<esphome PR number goes here>

## Checklist:

  - [ ] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.

### CODE REVIEW

File: devices/sonoff.rst [hunk 1]
```diff
@@ -309,6 +309,18 @@ Shelly 2
     GPIO5, Relay #2,
     GPIO12, SW Input #1,
     GPIO14, SW Input #2,
+    
+Shelly RGBW2
+--------
```
Comment (OttoWinter) [inline devices/sonoff.rst]:
```suggestion
------------
```

File: devices/sonoff.rst [hunk 2]
```diff
@@ -309,6 +309,18 @@ Shelly 2
...
+
+.. pintable::
+
+    GPIO12, LED Red,
+    GPIO15, LED Green,
+    GPIO14, LED Blue,
+    GPIO04, LED White,
+    GPIO??, SW Input,
+    GPIO??, Power Meter,
```
Comment (OttoWinter) [inline devices/sonoff.rst:323]: Do not add information here if we don't know the actual pin.

Also, the power meter chip on the Shelly is not supported - add a line below saying that power metering is not supported at this point.

Comment (erdii) [reply devices/sonoff.rst:323]: So do I either remove the unknown pins or add information about the unsupported power meter?

Comment (OttoWinter) [reply devices/sonoff.rst:323]: Both would be best :)  (so remove the unknown pins and add a quick comment below as for the power meter chip not being supported)

**Pull request closed without merging**